### ISSUE TITLE
Highfive stopped commenting test failures

### ISSUE THREAD

**Issue by ConnorGBrewster** [servo/highfive#141]
See:
servo/servo#11809
servo/servo#11792
servo/servo#11800
servo/servo#11798

**Comment by ConnorGBrewster** [comment]
Possibly fixed?
#11809

**Comment by KiChjang** [comment]
I'm thinking that this might be intermittent.

**Comment by jdm** [comment]
Yeah, it might be highfive hitting the API limits again. We should find a way to log when that occurs.
